jekyll/app/_scss/ie.scss

```diff
@@ -0,0 +1,25 @@
+body {
+  .page-books .wrapper .books-list .letter-wrapper {
+    clear: both/9;
```

Comment:
Since we load this css only on IE <= 9, theres no need for the "/9" hack (Or is there a reason?).